### PR TITLE
Db migration

### PR DESCRIPTION
**Pull request by kosiakkatrina**
**What**
Change field names and types to more closely resemble CORE data.
For checkbox answers, pass through "Yes"/"No" instead of true/false from the form in order to map it using enums.

**Why**
To replicate CORE schema and make it easier to run existing queries.

**Anything else**
Display checkbox answers

### CODE REVIEW

File: app/models/case_log.rb [hunk 1]
```diff
@@ -47,6 +48,60 @@ class CaseLog < ApplicationRecord
 
   enum status: { "not_started" => 0, "in_progress" => 1, "completed" => 2 }
 
+  enum ethnic: DbEnums.ethnic
```
Comment (baarkerlounger) [inline app/models/case_log.rb:51]: Could we put the enums into the module as well or do they have to be here?

**Review by baarkerlounger — APPROVED** [review]
Great stuff!

File: app/models/case_log.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/models/case_log.rb:51]: I don't think we can put it in module, it's something to do with being part of ActiveModel

File: app/controllers/case_logs_controller.rb [hunk 2]
```diff
@@ -107,7 +107,7 @@ def responses_for_page(page)
 
       if question_info["type"] == "checkbox"
         question_info["answer_options"].keys.reject { |x| x.match(/divider/) }.each do |option|
-          result[option] = question_params.include?(option)
+          result[option] = question_params.include?(option) ? "Yes" : "No"
```
Comment (baarkerlounger) [inline app/controllers/case_logs_controller.rb:110]: This looks a little weird. It won't work for actual boolean fields (which will save "No" as `true`, and probably not for integer fields either?) are checkbox questions consistently stored as strings? Looks like some are ints and some booleans from the schema?

Comment (kosiakkatrina) [reply app/controllers/case_logs_controller.rb:110]: Ah, just noticed the boolean checkboxes, they should really be integer. I'll fix that

**Pull request merged**